File: days_mobile/lib/models/Course.dart
class Course {
  int id;
  String name;
  String slug;
  int userId;

  Course(
      {this.id,
      this.name,
      this.slug,
      this.userId,});

  Course.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    name = json['name'];
    slug = json['slug'];
    userId = json['user_id'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['name'] = this.name;
    data['slug'] = this.slug;
    data['user_id'] = this.userId;

    return data;
  }
}

File: days_admin/app/Models/Course.php
<?php

namespace App\Models;

use App\Http\Traits\WithImageUpload;
use Illuminate\Database\Eloquent\Model;

class Course extends Model
{
    use WithImageUpload;

    protected $table = 'courses';

    protected $fillable = ['name', 'slug'];

    protected $dates = ['created_at'];

    public function user()
    {
        return $this->belongsTo('App\Models\User', 'user_id');
    }
}
